monster-cards/src/widgets/card/card-sprite-large.styl

```diff
@@ -0,0 +1,147 @@
+&.Cards-Momentum14 {
+	background-position: -0px -0px;
+}
```

Comment:
Sounds good, I used `sprity` to generate the sprite, when we get the proper images etc.. from Torrey we can refactor, ALTHOUGH, as it's an automated process and the outputted `CSS` is the same regardless, i'm personally not too bothered